.jenkins/JenkinsfileTestLinuxRelease

```diff
@@ -0,0 +1,132 @@
+
+// The below timeout is set in minutes
+GLOBAL_TIMEOUT = 240
+
+DOCKER_REGISTRY = "dcapdockerciregistry.azurecr.io"
+
+
+// Tests running on hardware with custom path to libdcap_quoteprov.so
+def ACCTest(String label, String version, String compiler, String build_type) {
+    stage("${label} ${compiler} SGX1-FLC ${build_type}") {
+        node("${label}") {
+            timeout(GLOBAL_TIMEOUT) {
+			
+                cleanWs()
+                checkout scm
+				
+                def dcap = load pwd() + "/.jenkins/src/Dcap.groovy"
+
+                // Clone the Open Enclave repo
+                dir("${WORKSPACE}/openenclave") {
+                    git url: 'https://github.com/Microsoft/openenclave.git'
+                }
+
+                // Run hardware tests using the libdcap_quoteprov.so build
+                def task = """
+                        cd ${WORKSPACE}/src/Linux
+                        ./configure
+                        make
+                        mkdir -p ${WORKSPACE}/openenclave/build
+                        cd ${WORKSPACE}/openenclave/build
+                        git submodule update --recursive --init 
+                        cmake ${WORKSPACE}/openenclave -G Ninja -DCMAKE_BUILD_TYPE=${build_type}
+                        ninja -v
+                        LD_LIBRARY_PATH=${WORKSPACE}/src/Linux ctest --output-on-failure
+                       """
+                dcap.Run(compiler, task)
+            }
+        }
+    }
+}
+
+// Test using oetools-test Docker image with /dev/sgx mounted inside container
+def ACCContainerTest(String label, String version) {
+    stage("Ubuntu ${version} Non-Simulation Container SGX1-FLC RelWithDebInfo") {
+        node("${label}") {
+            timeout(GLOBAL_TIMEOUT) {
+			
+                cleanWs()
+                checkout scm
+				
+                def dcap = load pwd() + "/.jenkins/src/Dcap.groovy"
+
+                // Clone the Open Enclave repo
+                dir("${WORKSPACE}/openenclave") {
+                    git url: 'https://github.com/Microsoft/openenclave.git'
```

Comment:
Nit. This github string and the string at line 51 "/.jenkins/src/Dcap.groovy" are reused a fair bit. Suggestion to move them to constants.

Reply:
This is not a constant. What it do is to find the path of the current dir. In different agents the paths may be different. It cannot be changed as constant.

Reply:
The hardcoded strings can be different in different agents? I'm referring to the hardcoded strings that are re-used.  My suggestion was making those local vars or something re-usable so you didn't have to define it 3+ times. Anyway just a suggestion.